cron/loginstats.go

```diff
@@ -0,0 +1,70 @@
+package cron
+
+import "database/sql"
+
+type Login struct {
```

Comment:
This type could use a better name too, probably `LoginCount` or `LoginStats` (so we could in the future add other stats to the struct) or similar.